hnn_core/gui/_viz_manager.py

```diff
@@ -501,13 +540,15 @@ def _on_plot_type_change(new_plot_type):
     plot_button.on_click(
         partial(
             _plot_on_axes,
-            widgets_simulation=simulation_selection,
+            simulations_widget=simulation_selection,
             widgets_plot_type=plot_type_selection,
-            target_simulations=target_data_selection,
+            data_widget=target_data_selection,
             spectrogram_colormap_selection=spectrogram_colormap_selection,
-            dipole_smooth=dipole_smooth,
+            dipole_smooth=simulation_dipole_smooth,
```

Comment:
Move the `dipole_smooth` keyword below `max_spectral_frequency`.

Reply:
Done